autosubmit/platforms/paramiko_platform.py

```diff
@@ -37,15 +56,59 @@ def wrapper(*args, **kwargs):
 
     return wrapper
 
-# noinspection PyMethodParameters
-class ParamikoPlatform(Platform):
```

Comment:
The class is still here, just further down :point_down: , after these functions.